packages/firebase_auth/firebase_auth_web/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   http_parser: ^4.0.0
-  intl: ^0.17.0
+  intl: ^0.18.1
```

Comment:
Shouldn't this be `^0.18.0` for compatibility with Flutter? I believe their version is set to `0.18.0`.

Reply:
[True](https://github.com/flutter/flutter/blob/stable/packages/flutter_localizations/pubspec.yaml), I'll downgrade to 0.18.0. It would have still resolved properly.